Python/Karger's Algorithm (Min Cut)/kargers.py

```diff
@@ -0,0 +1,92 @@
+import random
```

Comment:
Please don't create separate folder for the algorithm, instead just include the Program file under language's Graph folder with README.md content commented in program file

Reply:
Okay